### PR TITLE
clio-27937 th-error

### PR DESCRIPTION
**Pull request by JefCurtis**
- Add `th-error` component with tests. 
- Replace `th-lazy`'s error handling with `th-error`.

### CODE REVIEW

File: themis_components/thError/thError.directive.coffee [hunk 1]
```diff
@@ -0,0 +1,21 @@
+angular.module("ThemisComponents")
+  .directive "thError", ->
+    restrict: "E"
+    transclude: true
+    template: """
```
Comment (PANCAKES) [inline themis_components/thError/thError.directive.coffee]: Were we moving the templates to their own file or are we not enforcing that yet?

Comment (JefCurtis) [inline themis_components/thError/thError.directive.coffee]: No problem, I can move it.

File: themis_components/thError/index.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.th-error-container {
```
Comment (andreisoare) [inline themis_components/thError/index.scss]: You should remove the CSS from `th-lazy`, otherwise we will duplicate this entire CSS.

Comment (JefCurtis) [inline themis_components/thError/index.scss]: Done. thanks!

File: themis_components/thError/readme.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+### Markup
+```
+<th-error></th-error>
+```
+
+Pass in transcluded elements (will be effected by the same styling as default message)
+
+```
+<th-error><p>transcluded message</p></th-error>
```
Comment (seanhealy) [inline themis_components/thError/readme.md]: Do I need a `<p>` around my error message?

Also, the word transcluded here is a bit unnecessarily technical and speaks to us more than to the function of the component.

Comment (JefCurtis) [inline themis_components/thError/readme.md]: updated to make it (hopefully) clearer.

**Pull request merged**